engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -283,6 +283,19 @@ keep the image size down. Since the `RUN` statement starts with
 > **Note**: The official Debian and Ubuntu images [automatically run `apt-get clean`](https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105),
 > so explicit invocation is not required.
 
+#### Using pipes
+
+Where your `RUN` command depends on pipes, for example:
+
+
+	RUN wget -O - https://some.site | wc -l > /number
```

Comment:
Please use the triple-tick fencing syntax, with `Dockerfile` hint, instead of the 4-space indentation.